src/controllers/userController.js

```diff
@@ -107,6 +109,13 @@ const User = {
     const receiveEmails = (subscription === 'subscribe');
     await UserService.manageUserSubscription(id, receiveEmails);
     sendResult(res, 200, `you have been ${subscription}ed successfully`);
+  },
+
+  async logout(req, res) {
+    const data = { isLogged: false };
+    const condition = { id: req.user.userId };
+    await UserService.updateUser(data, condition);
+    sendResult(res, 200, 'Logout successful');
```

Comment:
- We just don't need to know that the user is logged out. We also need to know when the user  **_last logged out_**.
- The column name should be something like - `lastLoggedOutAt`, should default to null, which means that the user is logged out.